autoload/maktaba/json.vim

```diff
@@ -12,6 +12,164 @@ if !exists('maktaba#json#NULL')
       \ 'null': s:NULL, 'true': s:TRUE, 'false': s:FALSE}
 endif
 
+" Python implementation:
+
+" Initialize Vim's Python environment with the helpers we'll need.
+function! s:InitPython() abort
+  python << EOF
```

Comment:
Done. Note that this takes initialisation from ~6.5ms to ~9.3ms (on my machine).